### ISSUE TITLE
use of several ESP32's - 2nd chapter

### ISSUE THREAD

**Issue by sgrizzi** [sinricpro/esp8266-esp32-sdk#122]
Hi,
I am now trying to get up and running the second prototype of my ESP32 module which is supporting 18 devices.
The two prototypes are identical apart from the access credentials and deviceIDs.
Both ESP32 are connected to the same WiFi network.

As per your recommendation (see below), I activated a new app key/secret and used it for the new module.
In addition, moving forward one step at a time, to start with, I activated just one new doorbell device in the second ESP32 module.

I noticed the following:
-my Alexa home system immediately acknowledged the addition of the new device. Is this correct, even if I am using different app key/secret combinations?  If the second ESP module is physically away and connected to another WiFi network, would it still be registered under "my" Alexa system? Should I register in Sinric with a completely separated personal account to keep the second module disconnected from my Alexa system?
-the new doorbell device is created as just another member of my set of devices. Are there any means to group/link this ID to a specific app key/secret pair?
-in a nutshell, if tomorrow I have 10 or 20 identical ESP modules, belonging to different customers, and most probably delivered pre-configured (apart from the wifi credentials), how can I make sure that they can live "independently" and connect to their own Alexa's?

Best regards, Gabriele

You _can_ use the same credentials on multiple devices. For the future, however, there will be changes that will no longer allow this. I therefore recommend to use a separate **App**Key's for each **app**lication (ESP-Module).

_Originally posted by @sivar2311 in https://github.com/sinricpro/esp8266-esp32-sdk/issues/118#issuecomment-751847189_

**Comment by kakopappa** [comment]
Hi Gabriele, 

1. When you create a device in Sinric Pro account, it sends the newly added device details to Alexa regardless of the appkey/secret.

2. Each device is associated with its own secret, you select this when creating a device. 

3. You going to need to maintain different Sinric Pro accounts.  If you are building a commercial product, you should consider using our PaSS solution.

**Comment by sgrizzi** [comment]
Hi Aruna,
all clear. I am planning to set-up 2-3 additional prototypes to give them to other people next week for further testing.
If the outcome is positive -and I am reasonably confident- then indeed we will need to move from a prototype to making a "product".
I am definitely interested in discussing the details of your PaSS solution. I would like to understand more in detail how the commissioning process works and what additional sw integration is needed - if any.
What channel can we use to communicate on this? I would not like to use an open forum. These details are also relevant for our internal decisions, so the sooner we discuss this, the better.
Best regards, Gabriele
